src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__watch {
+    position: relative;
+    height: $watch-size;
+    width: $watch-size;
```

Comment:
i think we have to move width and height to block stop-watch. because when we move block to another place - our watch will be without these properties